app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from __future__ import annotations
+from typing import Optional
+
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: Optional[list[Player]]) -> int:
+    return sum([player.get_rating() for player in players])
```

Comment:
Do not pass redundant list to `sum`, refer to the [checklist](https://github.com/mate-academy/py-elves-and-dwarves/blob/master/checklist.md).